addons/sale_timesheet/tests/test_project.py

```diff
@@ -97,3 +97,55 @@ def test_fetch_sale_order_items(self):
         self.project_global.allow_billable = False
         self.assertFalse(self.project_global._get_sale_order_items())
         self.assertFalse(self.project_global._get_sale_orders())
+
+    def test_analytic_account_balance(self):
+        """
+            1) Add new billable project
+            2) Add Employee/SOL mapping in the project
+            3) Add Task and Timesheet with the same user
+            4) Assert analytic_account_balance is calculated
+        """
+        partner = self.env['res.partner'].create({
+            'name': 'Partner',
+        })
```

Comment:
You could use a partner created in the common class that we inherit, I think we have those partner: <https://github.com/odoo/odoo/blob/f34d8632572dfe8911b62471882eae60f6e6ae72/addons/account/tests/common.py#L143-L159>